src/app/retrospective/retrospective-feedback/retrospective-feedback.component.ts

```diff
@@ -388,7 +388,14 @@ export class RetrospectiveFeedbackComponent implements OnInit, OnChanges, OnDest
                 field: 'AssigneeID',
                 minWidth: 160,
                 editable: editable,
-                valueFormatter: (cellParams) => {
+                cellRenderer: (cellParams) => {
```

Comment:
Since both functions are the same, we should make a common function for the same, `cellRenderer` and `keyCreator`.

Reply:
done